template/src/main/twirl/ch.epfl.scala.index.views/artifact.scala.html

```diff
@@ -38,44 +38,56 @@
-          <li class="col-md-6 col-sm-6"><i class="fa fa-tag"></i> @project.artifacts.find(_.reference == artifact).map(_.releases.size).getOrElse(0) Releases</li>
-          <li class="col-md-6 col-sm-6"><i class="fa fa-sitemap"></i> @releases.find(rel => version.contains(rel.reference.version)).map(_.uniqueOrderedReverseDependencies.size).getOrElse(0) Dependents</li>
-        </ul>
+@statisticBlock(githubInfo: Option[GithubInfo]) = {
+
+  @githubInfo.map { github =>
+    <div class="statistic box">
+      <h4>Statistics</h4>
+      <div class="row">
+        <div >
+          <ul class="row">
+            <li class="col-md-6 col-sm-6"><i class="fa fa-eye"></i> @github.watchers watchers</li>
+            <li class="col-md-6 col-sm-6"><i class="fa fa-users"></i> @github.contributors.size Contributors</li>
+            <li class="col-md-6 col-sm-6"><i class="fa fa-star"></i> @github.stars Stars</li>
+            <li class="col-md-6 col-sm-6"><i class="fa fa-code-fork"></i> @github.forks Forks</li>
+            <li class="col-md-6 col-sm-6"><i class="fa fa-history"></i> @github.commits Commits</li>
+            <li class="col-md-6 col-sm-6"><i class="fa fa-exclamation-circle"></i> @github.issues Open issues</li>
+            <li class="col-md-6 col-sm-6"><i class="fa fa-tag"></i> @project.artifacts.find(_.reference == artifact).map(_.releases.size).getOrElse(0)
+              Releases</li>
+            <li class="col-md-6 col-sm-6"><i class="fa fa-sitemap"></i> @releases.find(rel => version.contains(rel.reference.version)).map(_.uniqueOrderedReverseDependencies.size).getOrElse(0)
+              Dependents</li>
+          </ul>
+        </div>
       </div>
     </div>
-  </div>
+  }.getOrElse("")
 }
 
 @** Contributor block *@
-@contributorsBlock(github: GithubInfo) = {
+@contributorsBlock(githubInfo: Option[GithubInfo]) = {
+
+  @githubInfo.map { github =>
     @if(github.contributors.nonEmpty) {
       <div class="collaborators box">
-        <h4>Contributors</h4>
+        <h4>Contributors (@github.contributors.size)</h4>
 
         <ul>
-          @for(contributor <- github.contributors) {
-            <li>
-              <a href="@contributor.url.target" target="_blank">
-                <img data-toggle="tooltip" data-placement="bottom" width="32" title="@contributor.login" src="@contributor.sizedAvatarUrl(32)" alt="">
-              </a>
-            </li>
-          }
+        @for(contributor <- github.contributors.take(100)) {
+          <li>
+            <a href="@contributor.url.target" target="_blank">
+              <img data-toggle="tooltip" data-placement="bottom" width="32" title="@contributor.login" src="@contributor.sizedAvatarUrl(32)" alt="">
+            </a>
+          </li>
+        }
+
         </ul>
+        @if(100 < github.contributors.size) {
+
+          <span>There are another @{github.contributors.size - 100} contributors</span>
```

Comment:
maybe a link to the contributor page on github ?

ex: https://github.com/apache/spark/graphs/contributors